data/json/items/armor/storage.json

```diff
@@ -2974,8 +2974,8 @@
     "id": "long_duffelbag",
     "type": "ITEM",
     "subtypes": [ "ARMOR" ],
-    "name": { "str": "longarm bag" },
```

Comment:
I think it's better to keep this as a rifle bag and to make a huge duffel bag a separate item. Though I'm honestly not too sure how common 140L bags would even be

Reply:
https://deepbluesub.com/en_us/product/c4-speargun-bag-single-150/

https://www.decathlon.co.uk/p/sheath-hunting-rifle-150cm-green/_/R-p-10515

There are a few more brands